app/src/main/java/com/github/se/polyfit/ui/screen/IngredientScreen.kt

```diff
@@ -43,7 +43,6 @@ fun IngredientScreen(
 
   fun goBackAndReset() {
     navigateBack()
-    mealViewModel.reset()
   }
 
   Scaffold(
```

Comment:
```suggestion
  Scaffold(
    topBar = { TopBar({ navigateBack }) }
```

can just remove the goBackAndReset fn